components/clm/src/biogeophys/SoilHydrologyType.F90

```diff
@@ -550,12 +550,12 @@ subroutine InitCold(this, bounds)
                  micro_sigma(c)/sqrt(2.0*shr_const_pi)*exp(-d**2/(2.0*micro_sigma(c)**2))         
             this%h2osfc_thresh_col(c) = 1.e3_r8 * this%h2osfc_thresh_col(c) !convert to mm from meters
          else
-            this%h2osfc_thresh_col(c) = 0._r8
+            this%h2osfc_thresh_col(c) = 0._r8     !changed from 0 to 1 TAO 29/8/2018
          endif
 
-         if (this%h2osfcflag == 0) then 
-            this%h2osfc_thresh_col(c) = 0._r8    ! set to zero for no h2osfc (w/frac_infclust =large)
-         endif
+#if (defined MARSH)
```

Comment:
this was set to a high number to prevent the water from running off into a "spillover" column and getting lost from the system

Reply:
So does it need to be set to 1 for MARSH simulations? Or is that a change that you undid?

Reply:
I originally had it at 1, then after a meeting with Dan realized it needed to be much higher to allow for standing water

Reply:
My code has it set to 2E3